index.js

```diff
@@ -169,8 +165,7 @@
         var $button = $target.is('button') ? $target : $target.closest('button');
         var $firstButton = this.$buttons.first();
         var $lastButton = this.$buttons.last();
-        var $prevButton = $button.prevAll(this.options.buttonsSelector).first();
```

Comment:
The problem was here.
`$button.prevAll(this.options.buttonsSelector)` with a relative `this.options.buttonsSelector`by exemple `> button` can’t work.
So I change solution using index to know current, last and first position.